k8s/README.md

```diff
@@ -60,7 +60,8 @@ To use the development image for testing with specific version of VPP, see
       - `IP`: IP address to be attached to the main interface;
     - `OtherVPPInterfaces` (other configured interfaces only get IP address assigned in VPP)
       - `InterfaceName`: name of the interface;
-      - `IP`: IP address to be attached to the interface.
+      - `IP`: IP address to be attached to the interface;
+    - `Gateway`: IP address of the default gateway for external traffic, if it needs to be configured.
```

Comment:
shouldn't gateway be a child of MainVppInterface and OtherVPPInterfaces or are all vpp interfaces requirdd to be on the same network to expand bandwidth (not familiar with multiple VPP interface configurations)

Reply:
I think Gateway is not an interface-local setting, but a VPP-global setting. I can point to an IP behind any of the VPP interfaces (managed by OtherVPPInterfaces). But still, you should have only one default gateway (one default route) in the setup.

Reply:
right, thx